docs/earthfile/earthfile.md

```diff
@@ -1056,6 +1065,15 @@ Specifies the platform to build on.
 
 For more information see the [multi-platform guide](../guides/multi-platform.md).
 
+#### `--allow-privileged` (experimental)
+
+{% hint style='info' %}
+##### Note
+The `--allow-privileged` flag has experimental status. To use this feature, it must be enabled via `VERSION --allow-privileged-from-dockerfile 0.8`.
+{% endhint %}
+
+When the Dockerfile build context points to an earthly artifact reference (e.g. +some-other-build-context/*), the `allow-privileged` flag will allow `RUN` commands under the referenced earthly target to make use of the `RUN --privileged` option.
```

Comment:
it's probably better to use the same example target that we use in our hint that appears above.
```suggestion
When the Dockerfile build context points to an earthly artifact reference (e.g. `+mybuildcontext/mydata/*`), the `allow-privileged` flag will allow `RUN` commands under the referenced earthly target to make use of the `RUN --privileged` option.
```